src/main/java/slothingwaffler/Deadline.java

```diff
@@ -0,0 +1,24 @@
+package slothingwaffler;
```

Comment:
great that everything is organized into a package!

you can consider having sub-packages to get more detailed with the organization